aws/instrumentation_test.go

```diff
@@ -40,6 +40,10 @@ type mockSNS struct {
 
 	spiedSetSubscriptionAttributesInput   *sns.SetSubscriptionAttributesInput
 	stubbedSetSubscriptionAttributesError error
+
+	spiedDeleteTopicInput   *sns.DeleteTopicInput
+	spiedDeleteTopicOutput  *sns.DeleteTopicOutput
```

Comment:
this should be `stubbed`, not `spied`. Here's a quick read to get a primer on the vocabulary: https://gaboesquivel.com/blog/2014/unit-testing-mocks-stubs-and-spies/

Reply:
actually, you don't even seem to be using this field anywhere so just remove it

Reply:
I am using these in DeleteTopic function and on publisher_test.go